genesis-tools/genesis-populate/src/lib.rs

```diff
@@ -317,8 +317,8 @@ impl GenesisBuilder {
     }
 
     fn add_additional_account(&mut self, account_id: AccountId) -> Result<()> {
-        let testing_init_balance: Balance = 10u128.pow(30);
-        let testing_init_stake: Balance = 0;
+        let testing_init_balance = Balance::from_near(10u128.pow(6));
```

Comment:
```suggestion
        let testing_init_balance = Balance::from_near(1_000_000);
```

is much more readable.